plugins/node/opentelemetry-instrumentation-awslambda/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@opentelemetry/instrumentation-awslambda",
```

Comment:
I think the name should be 
```suggestion
  "name": "@opentelemetry/instrumentation-aws-lambda",
```
you are patching `aws-lambda` and those are 2 words anyway